### PR TITLE
Make the footer sticky when necessary

### LINKED ISSUE #3465: Push footer to bottom of page
Hi! I have an idea to improve the appearance of exercism.io
I think the site would look better if the footer was pushed to the bottom of the page. Screenshots to explain:

The current notifications page has a lot of whitespace below the footer:
screen shot 2017-04-17 at 4 30 16 pm

If we push the footer to the bottom:
screen shot 2017-04-17 at 4 32 05 pm

Thoughts? I did this by modifying the `margin-top` property on the footer but I realize this isn't the correct approach. I imagine the correct technique uses some type of CSS `float` property or something else.
screen shot 2017-04-17 at 4 34 14 pm

### PR DESCRIPTION
**Pull request by lukaszklis**
Fixes #3465, picked up @arumoy-shome’s work done in #3491.

Preview here: https://drive.google.com/file/d/0B6BKR27AYAAddzI2dThGYUtjRFU/view?usp=sharing (.webm file recorded by Awesome Screenshot for Chrome).

### CODE REVIEW

**Comment by kytrinyx** [comment]
@lukaszklis Thank you so much for taking the time to pick this one up! The screencap looks like exactly what we want.

One thing that I find very important when working on open source stuff is preserving credit when I'm building on other people's work. Are you familiar with the term "Carrying a pull request?". Jess Frazelle wrote about various ways of closing pull requests, including "The Carry" here: https://blog.jessfraz.com/post/the-art-of-closing/

In short it means:

- creating a branch off of the PR
- adding your commits on top of the previous ones
- creating a new PR and adding "Closes #XYZ" to automatically close their open PR, as well as "Fixes #PRQ" for the original issue, if there is one

Would you mind opening a new PR that preserves the credit for @arumoy-shome?

**Comment by lukaszklis** [comment]
Hey, of course, the only reason why I have opened a new PR the way I did is the fact I had to use a different approach to @arumoy-shome. I'm definitely not against discrediting anyone, so in case it would feel better for you and the original author, I can base my branch on his, and add revert commits for his changes.

WDYT?

PS. Thanks for the article, will definitely read it. ;)

**Comment by kytrinyx** [comment]
Oh, nice—if you used a different approach, we're all good. Thank you!
